helm/hpcc/templates/dali.yaml

```diff
@@ -77,6 +79,7 @@ data:
     version: 1.0
     dali:
 {{ toYaml (omit $dali "logging" "services") | indent 6 }}
+      dataPath: {{ include "hpcc.getVolumeMountPrefix" (dict "root" $ "name" ($daliStorage.plane | default "dalistorage-pvc") "path" "dalistorage" "me" $daliStorage) }}
```

Comment:
is dalistorage-pvc the name of the plane?  If so, that is confusing.

Reply:
that's not right, also not right in a couple of other places.
It will still end up defaulting to the correct path (shouldn't match an existing plane), but should be changed